_includes/tutorials/versioned-ktables/kstreams/markup/dev/short-answer.adoc

```diff
@@ -0,0 +1,21 @@
+You can run your application with link:https://www.confluent.io/confluent-cloud/tryfree/[Confluent Cloud].
+
+
+In the Kafka Streams application use a `VersionedKeyValueStore` when creating your `KTable`.  The [history] retention time shown is just
```

Comment:
delete `[history]`?

Reply:
This is to clear up any confusion with the term  `retention` and how it pertains how long brokers keep records